src/DataProtection/DataProtection/src/AuthenticatedEncryption/CngCbcAuthenticatedEncryptorFactory.cs

```diff
@@ -1,7 +1,10 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
UTF-8 BOM was changed? (Validate that one wasn't added inadvertently.)

Reply:
My editor adds it. I generally haven't found it useful to fight it.